packages/data/json/9.9.9.json

```diff
@@ -3,7 +3,8 @@
     "title": "Egress from Dwelling Units",
     "subtitle": "Volume II - 9.9.9.",
     "description": "Ensure your dwelling unit has exits or egress doors that pass code.",
-    "startingSectionId": "1_intro"
+    "startingSectionId": "1_intro",
+    "walkthroughTitle": "9.9.9. Egress from Dwelling Units"
```

Comment:
I'm copying the pattern for the `sectionTitle` Which includes it. I think since we are manually doing this in json and it only has a use case to display them together then we should just keep them together here.